Jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline
   parameters {
     booleanParam(name: 'RUN_TESTS', description: 'if false skip testing', defaultValue: true)
     booleanParam(name: 'RUN_BUILD', description: 'if false skip building', defaultValue: true)
-    booleanParam(name: 'RUN_DOWNSTREAM', description: 'if false skip downstream jobs', defaultValue: true)
+    booleanParam(name: 'RUN_DOWNSTREAM', description: 'if false skip downstream jobs', defaultValue: false)
```

Comment:
What about this change, was it consulted with Thomas?

Reply:
yes, all good.